jmix-flowui/flowui/src/main/java/io/jmix/flowui/xml/layout/support/ComponentLoaderSupport.java

```diff
@@ -358,9 +356,9 @@ public void loadAllowedCharPattern(HasAllowedCharPattern component, Element elem
                 context.getMessageGroup(), component::setAllowedCharPattern);
     }
 
-    public void loadStep(com.vaadin.flow.component.Component component, Element element, Context context) {
-        loaderSupport.loadResourceString(element, "step", context.getMessageGroup())
-                .ifPresent(stepString -> {
+    public Optional<Duration> loadStep(Element element) {
```

Comment:
Let's call it `loadDuration`. Who knows where else we can use this method.

Reply:
i.e. you want the attribute name to also be passed to the method?

Reply:
da